acceptance/testdata/secret/secret-org.txtar

```diff
@@ -0,0 +1,16 @@
+# Prepare organization secret, GitHub Actions uppercases secret names
+env2upper ORG_SECRET_NAME=$RANDOM_STRING
```

Comment:
**Food For Thought**

If this is the only place that casing matters, maybe we can inject an already upper cased env var so we don't increase the command surface area? No need to rush a decision on it though, happy to revisit as we understand usage more.

Reply:
This will matter in `gh variable`, too.  And technically there are places where certain things should be all lowercase.

Reply:
Sorry, I should have been clearer.

> If this is the only place that casing matters

to

> If upper or lowercasing an environment variable that is injected from Go is the only place that casing matters

Thus if we had `$RANDOM_STRING` (or `$RANDOM_STRING_L` /  `$RANDOM_STRING_LOWER and `$RANDOM_STRING_U` / `$RANDOM_STRING_UPPER`) then the commands go away.

But maybe we also would need to do the same for `$SCRIPT_NAME`, and that seems like it starts to get annoying.

Let's leave it as is.